src/lambda-helpers.ts

```diff
@@ -40,3 +40,11 @@ export const getGithubOpts = () => ({
     ).toString("base64")}`,
   },
 });
+
+const twitterBearerToken = process.env.TWITTER_BEARER_TOKEN || "";
```

Comment:
we're going to want to add this environment variable to the github action and I'll need to add the secret to the repo:

https://github.com/dvargas92495/roam-js-extensions/blob/master/.github/workflows/master_lambda.yaml#L8-L13

Reply:
done

Reply:
looking at the docs, it looks like we have to generate a bearer token on each request based on a consumer key and secret? https://developer.twitter.com/en/docs/authentication/oauth-2-0

Reply:
Thanks for this comment :) 

I couldn't find an explicit mention that they might expire and we'd have to regenerate them. This thought crossed my mind too, but this [doc](https://developer.twitter.com/en/docs/authentication/oauth-2-0/bearer-tokens) says `You can also obtain a Bearer Token from the developer portal inside the keys and tokens section of your App's settings.`, so I assumed they were static. 

I'll read up more on oauth bearer tokens and get back to you.

Reply:
https://developer.twitter.com/en/docs/authentication/faq states that access tokens are not explicitly expired. So, while that shouldn't be a problem, I think it's a better approach to store the consumer Key and Secret and get the access token before making a request. That way even if we have to invalidate the bearer token, we need not make changes to the application.

I've pushed a commit with the above changes. 

Noting that since I can't really test the lambda locally + I can't call twitter API directly because of the CORS restrictions, the lambda code is essentially untested!